cypress/e2e/pages/address/[address].spec.ts

```diff
@@ -197,16 +194,22 @@ context("/address/[address] on mobile", () => {
         "Balances"
       );
 
-      cy.findAllByTestId("AddressTokenCard").within(() => {
-        cy.findByTestId("AddressTokenCard.TokenSymbol").should("be.visible");
-        cy.findByTestId("AddressTokenCard.TokenName").should("be.visible");
-        cy.findByTestId("AddressTokenCard.Type")
-          .should("be.visible")
-          .contains(/^[A-Z]{3}$/);
-        cy.findByTestId("AddressTokenCard.Amount")
-          .should("be.visible")
-          .contains(/\d+.\d+/);
-      });
+      cy.findAllByTestId("AddressTokenCard", { timeout: 15000 }).each(
+        ($tokenCard) => {
+          cy.wrap($tokenCard).within(() => {
+            cy.findAllByTestId("AddressTokenCard.TokenSymbol").should(
+              "be.visible"
```

Comment:
Can we make this into a single line like this `.should("be.visible");` to maintain its readability